common/src/test/scala/uk/ac/wellcome/test/fixtures/ServerFixtures.scala

```diff
@@ -1,28 +0,0 @@
-package uk.ac.wellcome.test.fixtures
-
-import com.twitter.finatra.http.EmbeddedHttpServer
-import com.twitter.inject.server.Ports
-import org.scalatest.Suite
-
-trait ServerFixtures { this: Suite =>
```

Comment:
In the end, I found it easier to split this back out than find somewhere for it to live.